plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
```

Comment:
- [ ] Throw an error during propagation when version of resource is not an existing commit
      sed -i 's/version: .*$/version: ABCD/g' interaction/softwares/roles/davical/meta/plasma.yaml
      compose and sync
``` 
ERROR   HEAD version of interaction/softwares/roles/davical/meta/plasma.yaml doesn't match real version
>
ERROR   version of interaction/softwares/roles/davical/meta/plasma.yaml doesn't match any existing commit
```

Reply:
+

Reply:
ERROR   version of interaction/softwares/roles/davical/meta/plasma.yaml doesn't match any existing commit